Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift

```diff
@@ -309,11 +356,14 @@ private enum ChangeTierTrackingType: TrackingViewNameProtocol {
             return .init(describing: EditDeductibleScreen.self)
         case .compareTier:
             return .init(describing: CompareTierScreen.self)
+        case .info:
+            return "Addon Info"
```

Comment:
don't we have any screen we can use as for the ones above? otherwise it's fine

Reply:
Yeah, I saw you upgraded "Info view" with some markdown logic. Will probably merge it later with it. Now I am just creating adhoc view for it